src/bp-templates/bp-nouveau/includes/template-tags.php

```diff
@@ -1320,6 +1325,7 @@ function bp_nouveau_get_nav_count() {
 	$bp_nouveau = bp_nouveau();
 	$nav_item   = $bp_nouveau->current_nav_item;
 	$count      = 0;
+	$coursesSlug     = apply_filters( 'bp_ld_sync/courses_group_tab_slug', 'courses' );
```

Comment:
@shohelbuddyboss Can you follow the naming convention of the variable name of WP `coursesSlug` should be a `courses_slug` ? Can you make the change please? and also make the tabbing proper here?